view/headerbar.twig

```diff
@@ -68,7 +68,7 @@
               {%- if term %}&q={{ term }}{% endif -%}
               {%- if vocabs %}&vocabs={{ vocabs }}{% endif -%}
               "
-              class="lang-button lang-button-all">
+              class="lang-button" id="lang-button-all">
```

Comment:
Thanks! I remember seeing other cases where Bootstrap (or plug-ins) re-wrote the DOM structure or attributes :disappointed: 

Sorry for yet another post-bootstrap upgrade. Hopefully we are getting close to a release now, with no more issues like this :crossed_fingers:

Reply:
I think a few more patches will be needed...there is still a small backlog of reports from users currently testing dev.finto.fi - mostly minor layout issues such as spacing and font sizes. I will open issues for bigger functional problems, but for minor or purely layout fixes it's simpler to just make a fix and open a PR.